tests/unittests/graphOptzTest.cpp

```diff
@@ -872,3 +872,49 @@ TEST_F(GraphOptz, concatElim) {
   // Check that the concat node is gone.
   EXPECT_EQ(countNodeKind(F_, Kinded::Kind::ConcatNodeKind), 0);
 }
+
+// Check the transformation Concat(Reshape(x) * N) -> Reshape(Concat(x * N)).
+TEST_F(GraphOptz, concatReshapes) {
+  const size_t shape[] = {10, 20};
+  const size_t reshape[] = {10, 1, 20};
+  Type t(ElemKind::FloatTy, shape);
+  std::vector<NodeValue> inputs;
```

Comment:
nit: since you know the result size in advance it's recommended to preallocate that space using one of the vector's ctors.

Reply:
changed to llvm::SmallVector<NodeValue, 10>